include/dxc/HLSL/LiveValueAnalysis.h

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Just a reminder that we discussed putting the license here.

Reply:
Adding.